src/app/component/auth/components/sign-in/sign-in.component.html

```diff
@@ -25,7 +25,7 @@ <h2>{{'user.auth.sign-in.fill-form' | translate}}</h2>
              name="email"
              [(ngModel)]="userOwnSignIn.email"
              (input)="configDefaultErrorMessage()"
-             [ngClass]="emailControl.invalid && (emailControl.dirty || emailControl.touched ) || backEndError?
+             [ngClass]="emailControl.invalid && (emailControl.dirty || emailControl.touched ) || backEndError || emailErrorMessageBackEnd?
                 'alert-email-validation':'successful-email-validation'"
```

Comment:
Please, separate logical && and || in different lines for better code readable

Reply:
Fixed